mangaki/mangaki/management/commands/add_anidb.py

```diff
@@ -0,0 +1,25 @@
+from django.core.management.base import BaseCommand, CommandError
+from mangaki.utils.anidb import AniDB
+from mangaki.models import Work, Category
+from django.db.models import Count
+from urllib.parse import urlparse, parse_qs
```

Comment:
Sure, theorically, [autoflake](https://pypi.python.org/pypi/autoflake/) should do the job.